src/components/BuyButton.tsx

```diff
@@ -31,7 +31,12 @@ const Button = styled.button<
 
 const stripePromise = loadStripe(`${process.env.REACT_APP_STRIPE_API_KEY}`);
 
-const BuyButton = () => {
+interface BuyButtonProps {
+  successUrl: string;
+  cancelUrl: string;
+}
+const BuyButton: React.FC<BuyButtonProps> = (props) => {
```

Comment:
Just had to have a little read up on what React.FC means - sounds like a terrible football team.

Reply:
But yeah, from my understanding, with an F.C the only thing that can be returned here is a string.. hope i've understood that correctly

Reply:
Oh it works like a dream btw